### ISSUE TITLE
Cloudflare crashes

### ISSUE THREAD

**Issue by NikolaRHristov** [nhristov/astro-compress#322]
https://twitter.com/leo40510/status/1769785561160593641?t=tXWVK29K2Sab54UNADyYVQ

```sh
generating optimized images 04:24:02.535

node: malloc.c:3344: __libc_malloc: Assertion `!victim || chunk_is_mmapped (mem2chunk (victim)) || ar_ptr == arena_for_chunk (mem2chunk (victim))' failed.
```

**Comment by NikolaRHristov** [comment]
Can you try the latest `astro-compress` [v2.2.21](https://github.com/Playform/CompressAstro/releases/tag/AstroCompress%2Fv2.2.21) and `@playform/compress-astro` [v0.0.2](https://github.com/Playform/CompressAstro/releases/tag/CompressAstro%2Fv0.0.2) ?

**Comment by NikolaRHristov** [comment]
Should be fixed in the latest `astro-compress` [v2.2.21](https://github.com/Playform/CompressAstro/releases/tag/AstroCompress%2Fv2.2.21) and `@playform/compress-astro` [v0.0.2](https://github.com/Playform/CompressAstro/releases/tag/CompressAstro%2Fv0.0.2)
